src/backend/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": false,
+    "noEmit": true,
+    "typeRoots": ["types", "node_modules/@types"]
+  },
+  "include": ["*.js", "test/**/*"],
+  "exclude": ["node_modules/"]
```

```suggestion
  "exclude": ["node_modules"]
```
I _think_ that's the correct syntax, at least that's what I've always used.